lib/network/call_account_api.js

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line import/order
+const config = require("../config");
+const utils = require("../utils");
+const ensureOption = require('../utils/ensureOption').defaults(config());
+const make_request = require('./make_request');
+
+const { ensurePresenceOf } = utils;
+
+function call_account_api(method, uri, params, callback, options) {
+  ensurePresenceOf({ method, uri });
+  const cloudinary = ensureOption(options, "upload_prefix", "https://api.cloudinary.com");
+  const account_id = ensureOption(options, "account_id");
+  const api_url = [cloudinary, "v1_1", "provisioning", "accounts", account_id].concat(uri).join("/");
+  const auth = {
+    api_key: ensureOption(options, "provisioning_api_key"),
```

Comment:
This is confusing, why don't you call `api_key` and `api_secret` just `key` and `secret`(or user/pass), to avoid ambiguous interpretation of the `cloudinary:` api_key and api_secret